cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -171,12 +172,17 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
           -- version that this node will declare that it understands, when it
           -- is in the Mary era. Since Mary is currently the last known
           -- protocol version then this is also the Mary protocol version.
+          --
+          -- During testing of the Alonzo era, we conditionally declare that we
+          -- know about the Alonzo era. We do so only when a config option for
+          -- testing development/unstable eras is used. This lets us include
+          -- not-yet-ready eras in released node versions without mainnet nodes
```

Comment:
Can a specially forged packet break a node?

Reply:
No. It's just about not prematurely advertising support for a new era.